source/runbooks/cloudwatch-networking-alarms.html.md.erb

```diff
@@ -0,0 +1,97 @@
+* NoVPCAttachmentTraffic
+* route-table-changes
+* vpc-changes
+
+### nacl-changes
+
+This alarm triggers when an API call is made to create, delete, or update a network access control list (NACL). Our
+NACLs are fairly static, but can be amended when customers feed in values for east/west traffic, or private access to
+other locations such as PSN endpoints.
+
+Should this alarm trigger you should confirm the NACL in question and find the cause of the alarm. You should check our
+[Github Actions](https://github.com/ministryofjustice/modernisation-platform/actions?query=workflow%3Acore-vpc-*) to see
+if any recent actions correspond to the triggered alarms. You can also investigate the CloudTrail logging for the
+relevant account.
+
+As we make all durable changes through code, this should show you the cause of the `nacl-changes` alert. It ought to be
+legitimate in all cases, but can sometimes be related to alarms such as `NoVPCAttachmentTraffic` where a NACL blocks
+traffic from exiting a VPC.
+
+It is unlikely to cause traffic to be blocked from entering a VPC as the transit gateway endpoint subnets have fixed
+permissive NACLs.
+
+### network-gateway-changes
+
+This alarm triggers when an API call is made to create, delete, or attach an Internet Gateway or Customer Gateway. As
+with how static our VPC configuration is, it is unlikely that this alarm will trigger unintentionally.
+
+Should this alarm trigger you should confirm the gateway in question and find the cause of the alarm. You should check
+our [Github Actions](https://github.com/ministryofjustice/modernisation-platform/actions?query=workflow%3Acore-vpc-*) to
+see if any recent actions correspond to the triggered alarms. You can also investigate the CloudTrail logging for the
+relevant account.
+
+As we make all durable changes through code, this should show you the cause of the `network-gateway-changes` alert. As
+with the `nacl-changes` it ought to be legitimate in all cases. It is likely to affect customer traffic coming into a
+service from the internet in the case of an Internet Gateway. In the case of a Customer Gateway this will be seen
+alongside a VPN tunnel moving to a `down` state.
+
+### NpVPCAttachmentTraffic
+
+This alarm triggers when no traffic has been seen traversing a VPC Transit Gateway attachment and exists on a per-VPC
+basis. The intent for this alarm was to alert us when a change or event occurred that caused a cessation of traffic from
+a VPC to the Modernisation Platform Transit Gateway.
+
+Should this alarm trigger you should confirm the VPC in question and assess the behaviour of traffic over the alarm
+period, extending past this alarm to observe what normal behaviour should look like. It is possible for this alarm to
+trigger during ordinary operation when the polling window is short; no traffic over five minutes is not uncommon.
+
+In the event that this alarm triggers, and it has been deemed legitimate, your next step should be to confirm that there
```

Comment:
What's the best way to know if it's legitimate?

Reply:
It's something to be judged on a case by case basis, but I'll put an example of a legitimate trigger and a false positive in if that helps?

Reply:
It would really help thank you!